pkg/analyzer/analyzers/mysql/mysql_test.go

```diff
@@ -0,0 +1,163 @@
+package mysql
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/analyzers"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/config"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
+)
+
+const (
+	mysqlPass = "23201da=b56ca236f3dc6736c0f9afad"
+	mysqlHost = "localhost"
+	mysqlPort = "3308" // Do not use 3307, as local dev environments can use it for other things
+
+	inactivePass = "inactive"
+	inactiveHost = "192.0.2.0"
```

Comment:
Highlighting the linter complaining that these are unused here.

Reply:
Thank you for highlighting it. It also made me realize that I was manually handling the container instead of using test-container library.